storage/src/vespa/storage/persistence/persistenceutil.h

```diff
@@ -43,20 +42,27 @@ class MessageTracker : protected Types {
      * commands like merge. */
     void dontReply() { _sendReply = false; }
 
-    api::StorageReply::SP getReply() {
-        return _reply;
+    bool hasReply() const { return bool(_reply); }
+    const api::StorageReply & getReply() const {
+        return *_reply;
+    }
+    api::StorageReply & getReply() {
+        return *_reply;
+    }
+    api::StorageReply::SP getReplySP() {
```

Comment:
I'd prefer this to be called something like `steal_reply_sp`, as the current name implies the function doesn't have any side effects

Reply:
Done.